components_new/asp/tests/DbConn.py

```
@@ -0,0 +1,29 @@
+import mysql.connector
+import os 
+
+def get_connection():
+    
+    isDocker = os.environ.get('IS_RUNNING_IN_DOCKER', False)
+
+    if isDocker:
+        host = "mariadb-container"
+    else:
+        host = "localhost"
```

Ruff (0.8.2)

Use ternary operator `host = 'mariadb-container' if isDocker else 'localhost'` instead of `if`-`else`-block

Replace `if`-`else`-block with `host = 'mariadb-container' if isDocker else 'localhost'`

(SIM108)